modules/ROOT/pages/anypoint-custom-metrics-connector.adoc

```diff
@@ -4,16 +4,43 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The Anypoint Custom Metrics connector enables you to send operational and business metric values from a hybrid application to Anypoint Monitoring, where the metrics can be visualized in real time using Custom Dashboards. 
+The Anypoint Custom Metrics connector enables you to send operational and business metric values from a hybrid application to Anypoint Monitoring, where the metrics can be visualized in real time using the built-in *Custom metrics* dashboard. 
 
-Once the metrics are in a Custom Dashboard, you can filter and group the metrics along the included dimensions. You can include multiple custom and built-in metrics in a single dashboard visualization.
+Once you select the metrics to display in the dashboard, you can filter and group the metrics along the included dimensions. You can include multiple custom and built-in metrics in a single dashboard visualization.
```

Comment:
Do you have to import the connector and configure it first? As a fairly new customer, I think of connectors as being a thing I have to configure/select/import and a dashboard as being part of the UI, so this paragraph confuses me a bit. Even the connectors shipped with Studio have to be configured...?